robot/description/urdf/casters.urdf.xacro

```diff
@@ -41,30 +40,38 @@
         <kd value="1.0" />
     </gazebo>
 
+    <gazebo>
+        <joint name='${tf_prefix}/front_caster_link_joint' type='ball'>
+           <parent>${tf_prefix}/chassis_link</parent>
+           <child>${tf_prefix}/front_caster_link</child>
+           <pose frame=''>0 0 0 0 0 0</pose>
+        </joint>
+    </gazebo>
```

Comment:
It would be nice to have a comment quickly explaining why a ball joint is (if it is short). Otherwise it would be good to have a comment explaining that this tag will override the joint type defined above (so the `front_caster_link_joint` will be of type `ball` and not `fixed`)

Reply:
I have removed the ball caster since they don't seem to be making a huge difference. I've tested with the following 3 configurations:
1) fixed only 
2) continuous (with axis = 1 1 1)
3) fixed w/ ball

Ultimately they all seem to give roughly the same performance.